### PR TITLE
fixed font color and weight in setting search bar

### LINKED ISSUE #658: [Bug]: Settings search bar font color and font weight do not look like win11
### What happened?

Inside settings, Search bar font color should be white and also font weight is not looking similar to win11.

Could you please assign me this issue.

image

### To reproduce the behavior?

reproduce!

### What browser are you seeing the problem on?

Chrome

### PR DESCRIPTION
**Pull request by rwiteshbera**
# Description
Fixed #658 
image

### CODE REVIEW

**Comment by yashash1511** [comment]
image

i can barely see it this way

File: src/containers/applications/apps/assets/settings.scss [hunk 1]
```diff
@@ -58,6 +58,8 @@
         border-radius: 4px;
         padding: 8px;
         margin: 8px 16px 20px;
+        color: white;
```
Comment (yashash1511) [inline src/containers/applications/apps/assets/settings.scss:61]: use var(--dark-txt) instead of white.
also font-weight: 500 not required.

**Comment by rwiteshbera** [comment]
Could you please elaborate me a little bit? The preview is working fine.

File: src/containers/applications/apps/assets/settings.scss (hunk 1 above)
Comment (rwiteshbera) [reply src/containers/applications/apps/assets/settings.scss:61]: Ok

**Comment by yashash1511** [comment]
> Could you please elaborate me a little bit? The preview is working fine.

yeah try typing anything in the search box in light mode.

**Comment by yashash1511** [comment]
@rwiteshbera once you sign the CLA we can merge this :)

**Comment by rwiteshbera** [comment]
> @rwiteshbera once you sign the CLA we can merge this :)

How to sign it?

**Comment by yashash1511** [comment]
> I have read the CLA Document and I hereby sign the CLA

just copy paste the above.

**Comment by rwiteshbera** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**